iquip/apps/builder.py

```diff
@@ -564,10 +565,12 @@ def scanArguments(self) -> Dict[str, Any]:
         Returns the arguments of the range scan.
         """
         return {
+            "ty": "RangeScan",
             "start": self.startSpinBox.value(),
             "stop": self.stopSpinBox.value(),
             "npoints": self.npointsSpinBox.value(),
-            "randomize": self.randomizeCheckBox.isChecked()
+            "randomize": self.randomizeCheckBox.isChecked(),
+            "seed": None
```

Comment:
As I said before, the feature to set the seed manually looks necessary, hence please leave this as an issue.